osgar/drivers/test_system_monitor.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+from unittest.mock import MagicMock
+import sys
+sys.modules['psutil'] = MagicMock()
+from osgar.drivers.system_monitor import get_timestamp, SystemMonitor
+
+
+class TestSystemMonitor(unittest.TestCase):
+
+    def test_get_timestamp(self):
```

Comment:
can you please test also just single buffer `b']'`?

Reply:
or even empty string?

Reply:
The last two commit should solve this issue. Definitely, however, the dmesg logging should be in a separate module.